roles/hello-world-apb/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
+1 to how you organized this. Good idea including defaults for what the broker provides